src/mono/mono/mini/interp/transform.c

```diff
@@ -403,6 +403,14 @@ create_interp_local_explicit (TransformData *td, MonoType *type, int size)
 
 }
 
+static void
+create_interp_dummy_var (TransformData *td)
+{
+	g_assert (td->dummy_var < 0);
+	td->dummy_var = create_interp_local_explicit (td, m_class_get_byval_arg (mono_defaults.void_class), 8);
+	td->locals [td->dummy_var].offset = 0;
```

Comment:
You should also mark it as global so the var offset allocator ignores it.

`td->locals [td->dummy_var].flags = INTERP_LOCAL_FLAG_GLOBAL;`